File: OmkarFCUBEAPP/src/app/models/ewaybillextlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Ewaybillextmodel } from "./ewaybillextmodel";

export class Ewaybillextlistmodel {
    ewaybillextList: Ewaybillextmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}

File: FCUBEAPI/Consignment/Models/EwayBillExtListModel.cs
using Shared.Models;

namespace Consignment.Models
{
    public class EwayBillExtListModel
    {
        public List<EwayBillExtModel> EwaybillextList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }

}

